org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/cleanup/CleanUpRegistry.java

```diff
@@ -75,7 +76,7 @@ public List<TextEdit> getEditsForAllActiveCleanUps(TextDocumentIdentifier textDo
 			return Collections.emptyList();
 		}
 
-		List<String> compilerOptsToEnable = cleanUpsToRun.stream() //
+		List<String> compilerOptsToEnable = cleanUpsToRun.stream().filter(cleanUp -> cleanUp != null) //
```

Comment:
I think it makes sense to do this filter on line 73, so that if there are only invalid clean up ids, then the check at line 75 fails, and we save time by not computing the AST.